src/Symfony/Component/Form/Tests/Extension/Core/EventListener/FixUrlProtocolListenerTest.php

```diff
@@ -20,10 +20,25 @@
 
 class FixUrlProtocolListenerTest extends TestCase
 {
+    /**
+     * @dataProvider provideUrlToFix
+     */
+    public function testFixUrl($data)
```

Comment:
I suggest avoiding to swap the test and its provider, to avoid merge conflicts between branches

Reply:
I'm going to deal with the conflicts :)